pages/api/sol/providers/index.ts

```diff
@@ -6,7 +6,19 @@ import {
 } from '@awell-health/sol-scheduling'
 import { getSolEnvSettings, API_ROUTES, API_METHODS } from '../utils'
 import { omit } from 'lodash'
-import { log } from '../../../../src/utils/logging'
+
+// Helper function to log messages to the API route
+const log = async (params: {}, severity: string, error: string | {} = '') => {
+  try {
+    await fetch('/api/log', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ params, severity, error }),
+    })
+  } catch (err) {
+    console.error('Error sending log request:', err)
+  }
+}
```

Comment:
This is a bit confusing. Given that the handler code executes server side, why do we need to post requests to the log API route instead of directly calling the underlying log function ?